2022q3/clusteradm.adoc

```diff
@@ -0,0 +1,35 @@
+=== Cluster Administration Team
+
+Links: +
+link:https://www.freebsd.org/administration/#t-clusteradm[Cluster Administration Team members] URL: link:https://www.freebsd.org/administration/#t-clusteradm[https://www.freebsd.org/administration/#t-clusteradm]
+
+Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
+
+FreeBSD Cluster Administration Team members are responsible for managing the machines the Project relies on to synchronise its distributed work and communications.
+In this quarter, the team has worked on the following:
+
+* Added additional storage to the CI system.
+It will help store more artifacts.
+* link:https://www.vuxml.org/freebsd/index.html[VuXML] deployed in all official mirrors.
+It speeds up the `pkg audit` functionality.
+* A new (and additional) monitoring system is in place.
+* A few old and faulty machines were decommissioned.
+* Moved several services to newer hardware.
+* Regular cluster-wide software upgrades
+* Regular support for FreeBSD.org user accounts
+* Regular disk and parts support (and replaces) over all physical hosts and mirrors.
+
+Work in progress:
+
+* git infra: Add `--filter` support.
+* Work with the PowerPC team to improve the package builders, universal, and reference machines.
+* Site audit at our primary site: inventory of spares and other miscellanea occupying space in our cabinets.
+* Discussions with link:https://www.juniper.net/[Juniper] about a donation of new switches for our primary site.
+* Plan for a large scale network upgrade at our primary site.
+* Cluster refresh (more extended project).
+Most cluster machines are running FreeBSD `13-STABLE` or `14-CURRENT` as of 2022-09-30.
+Only a handful of machines are still on FreeBSD `12-STABLE`.
+
+We are looking for an additional full mirror site (five servers) in Europe.
+See link:https://wiki.freebsd.org/Teams/clusteradm/generic-mirror-layout[generic mirrored layout] for our needs.
+Offers of additional single-server mirrors (see link:https://wiki.freebsd.org/Teams/clusteradm/tiny-mirror[tiny mirror]) are always welcome too, especially in Europe.
```

Comment:
I'd simplify that to "Offers of link:https://wiki.freebsd.org/Teams/clusteradm/tiny-mirror[additional single-server mirrors] are always welcome too, especially in Europe." for more link text context (better accessibility for screenreader users, who may be presented with a list of links separate from the main body of text).